### PR TITLE
chore: Update dependencies and code style

### PR DESCRIPTION
**Pull request by germanattanasio**
## Changes

* Update dependencies to mitigate critical security issues
* Update code to use single quote rather than a mix of single and double quote
* Added tests to make sure errors are reported

### CODE REVIEW

**Comment by christiancompton** [comment]
@germanattanasio Handlebars in already included as a core dependency in this generator:  https://github.com/ibm-developer/generator-ibm-core-node-express/pull/159/files#diff-2018087f584c4398b5c3a23fc0e5f9dbL20

**Comment by germanattanasio** [comment]
@christiancompton The dependency was not part of the `package.json`. I just added it and that fixed the problem. See https://github.com/ibm-developer/generator-ibm-core-node-express/pull/159/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15

**Comment by christiancompton** [comment]
@alembe Can you review this?  I do not see why this is needed in the package.json, since we include it in `/lib`.  None of our generators use handlebars in our package.json, because they all have it by default: https://github.com/ibm-developer/generator-ibm-core-node-express/blob/develop/lib/handlebars.js.  

What problem does this address?   Per my previous point, even if we pull in the handlebars module, we still are referring to what we use in `/lib`.

**Comment by germanattanasio** [comment]
@christiancompton The errors I was seeing where when using the `skit` to develop the starter kit locally. Even with the errors, it was still generating the right files, it was just annoying so I added the library to avoid the error.

**Comment by jkingoliver** [comment]
@germanattanasio you're using the skit tool to create a starter kit? That tool is not being maintained and was intended originally to be a POC/MVP and has a lot of issues.

**Comment by christiancompton** [comment]
I completely understand now.  I apologize - I thought `lib` contained the contents of the package and not just ancillary helpers.  These changes look good.  The handlebars changes have been addressed in #160.  We will wait until after THINK to process the cosmetic changes.

**Comment by germanattanasio** [comment]
I'm happy with continuing the cosmetic changes after Think. I just did the most obvious first, but there are a variety of good practices that could make this module easier to maintain

File: package.json [hunk 1]
```diff
@@ -11,13 +11,12 @@
     "lib"
   ],
   "dependencies": {
```
Comment (germanattanasio) [inline package.json]: I've updated some of the dependencies and removed `yosay` since it wasn't being used.

File: package.json [hunk 2]
```diff
@@ -31,14 +30,14 @@
...
     "coveralls": "^3.0.2",
-    "eslint": "^3.1.1",
+    "eslint": "^5.14.0",
     "eslint-plugin-ejs": "0.0.2",
     "mocha": "^5.2.0",
     "mocha-multi-reporters": "^1.1.7",
-    "nyc": "^13.1.0",
-    "standard-version": "^4.4.0",
+    "nyc": "^13.3.0",
+    "standard-version": "^5.0.0",
```
Comment (germanattanasio) [inline package.json]: Most of the libraries with major updates are because they drop Node 4

**Comment by germanattanasio** [comment]
I just added 3 tests to make sure errors are reported

**Pull request merged**